src/main/java/org/folio/circulation/support/Clients.java

```diff
@@ -247,5 +260,19 @@ private CollectionResourceClient createCalendarStorageClient(
     throws MalformedURLException {
     return getCollectionResourceClient(client, context, "/calendar/periods");
   }
-  
+
+  private CollectionResourceClient createPatronNoticePolicesStorageClient(
+    OkapiHttpClient client,
+    WebContext context)
+    throws MalformedURLException {
+    return getCollectionResourceClient(client, context, "/patron-notice-policy-storage/patron-notice-policies");
```

Comment:
Personally, I'd split this into two lines before the path, as otherwise it is quite a long line

Reply:
Agree, I'll split this.